docs/usage/about-us.md

```diff
@@ -0,0 +1,47 @@
+# About us
+
+Renovate was created by [WhiteSource](https://www.whitesourcesoftware.com/) staff and they continue to work on Renovate.
+
+More than 600 outside contributors helped improve Renovate.
+
+## Special thanks
+
+We want to highlight the work of these awesome people.
+Thank you for your time and effort!
+
+### Maintainers
+
+Renovate is maintained by:
+
+- [@rarkins](https://github.com/rarkins)
+- [@viceice](https://github.com/viceice)
+- [@JamieMagee](https://github.com/JamieMagee)
+
+### Maintainers for features
+
+Next up, we have these people who help maintain parts of Renovate:
+
+- [@HonkingGoose](https://github.com/HonkingGoose) master of the docs and community manager
+- [@zharinov](https://github.com/zharinov) focused on parsing, Gradle and Maven
+- [@secustor](https://github.com/secustor) worked on Terraform and Helm
+- [@fgreinacher](https://github.com/fgreinacher) worked on NuGet
+- [@Turbo87](https://github.com/Turbo87) has helped in multiple areas, especially Cargo
+
+### Valuable contributions
+
+We want to highlight these valuable contributions, even if they are one-offs.
+Some features made a lot of people happy, and efficient!
+
+- [@ikesyo](https://github.com/ikesyo) regularly helpful
+- [@astellingwerf](https://github.com/astellingwerf) reviews PRs
+- [@danports](https://github.com/danports) Worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR.
```

Comment:
All other sentences have a lower case letter after the username, and don't end with a full stop, so:
```suggestion
- [@danports](https://github.com/danports) worked on the Flux manager, and other managers. Feel free to ping `@danports` for any Flux-related issue or PR
```